packages/core/src/types/IImageCalibration.ts

```diff
@@ -0,0 +1,15 @@
+import CalibrationTypes from '../enums/CalibrationTypes';
+
+/**
+ * IImageCalibration is an object that stores information about the type
+ * of image calibration.
+ */
+export interface IImageCalibration {
```

Comment:
it is very odd that we have small casing and upper casing in this, maybe we should be consistent or if group the ones that need to be upper case in some other property

Reply:
please add documentation to each entry/property